webknossos-datastore/app/com/scalableminds/webknossos/datastore/models/datasource/InboxDataSource.scala

```diff
@@ -37,12 +37,12 @@ package object inbox {
 
   case class UnusableDataSource[+T <: DataLayerLike](id: DataSourceId,
                                                      status: String,
-                                                     scale: Option[Vec3Double] = None,
+                                                     scale: Option[VoxelSize] = None,
                                                      existingDataSourceProperties: Option[JsValue] = None)
       extends GenericInboxDataSource[T] {
     val toUsable: Option[GenericDataSource[T]] = None
 
-    val scaleOpt: Option[Vec3Double] = scale
+    val scaleOpt: Option[VoxelSize] = scale
```

Comment:
here again: Maybe call ist `voxelSizeOpt` as its type is VoxleSize. But I do also understand that the property name on a dataset json sent to a client still stays `dataset.scale` and thus `scaleOpt` also makes kinda sense. But I would prefer to have it callend `voxelSizeOpt` so that just the publicWrites methods needs to care about calling the json property sent to the client `scale`.

Fee free to argue :)

Reply:
I renamed it to voxelSizeOpt, I think that’s fair. I’ll go through more code and see which parts can be renamed easily without worrying about compatibility.